src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.cpp

```diff
@@ -42,6 +43,15 @@
 
 #include <QVBoxLayout>
 
+// When a non-nucleotide sequence is selected, the widget should be disabled. But the groupbox titles continue to be
+// black as if they enabled. This code makes them gray.
+static void makeGroupboxTittleGrayIfDisable(QGroupBox *const gb) {
```

Comment:
This function is cut off from everything, It's better to move it to either the `PCRPrimerDesignForDNAAssemblyOPWidget` class or the anonimus namespace. But, as far is this function is used in the only one function, you can just simply define it as lambda.